src/__tests__/ddl.test.ts

```diff
@@ -18,6 +18,24 @@ describe(`ddl`, () => {
     () => Promise.resolve({ rows: [], affectedCount: 0 }),
   );
 
+  it('should retrieve column definition info from column', () => {
+    const column = db.foo.id;
+    expect(column.getDefinition().getDefinition()).toMatchInlineSnapshot(`
```

Comment:
Why are there two chained `.getDefinition()` calls?

Reply:
Yeah this is a bit strange.

1. I added the ability for a `Column` to call `getDefinition()`, which returns a `ColumnDefinition`.
2. `ColumnDefinition` has a [function](https://github.com/Anrok/mammoth/blob/1e5705229b4200bf906fba099f5af87d427b1c4d/src/column.ts#L52) called `getDefintion()`, which returns a `ColumnDefinitionFormat`.

I also found it confusing that the latter's `getDefinition` function returns a `ColumnDefinitionFormat`.  It seems like this should be called `getDefinitionFormat`, but I didn't want to make that change in this PR since it would break backwards compatibility.